cni/network/invoker_azure.go

```diff
@@ -50,7 +51,9 @@ func (invoker *AzureIPAMInvoker) Add(nwCfg *cni.NetworkConfig, subnetPrefix *net
 	defer func() {
 		if err != nil {
 			if len(result.IPs) > 0 {
-				invoker.plugin.ipamInvoker.Delete(&result.IPs[0].Address, nwCfg, nil, options)
+				if er := invoker.plugin.ipamInvoker.Delete(&result.IPs[0].Address, nwCfg, nil, options); er != nil {
```

Comment:
Why change in this file?

Reply:
`invoker.plugin.ipamInvoker.Delete` can return an error and we were previously ignoring it completely
because parts of this file were changed, golangci-lint flagged this problem.

Reply:
this is using a tightly scoped variable `er` and wrapping the original error with `%w` in the return so it's not lost